src/en/juju-backups.md

```diff
@@ -0,0 +1,405 @@
+
+```bash
+cd ~
+tar -cpzf juju-client-$(date "+%Y-%b-%d_%H-%M-%S").tar.gz .juju 
+```
+
+This command datestamps the created file for easy identification, you may of
+course, call it what you wish.
+
+If you have used the local provider, you will need to run the command as a 
+superuser:
+
+```bash
+cd ~
+sudo tar -cpzf juju-client-$(date "+%Y%m%d-%H%M%S").tar.gz .juju 
+```
+
+This is because the local provider makes use of superuser privileges, and
+consequently some of the files belong to 'root' and cannot be read by a normal
+user. You will also find that the backup archives are substantially bigger. 
+As the local provider stores everything in '~/.juju', you are effectively 
+backing up the entire local environment also. If you aren't concerned about the 
+local user, you can omit the local provider files by supplying the 
+```--exclude-path=``` switch as many times as necessary and specifying the name 
+of your local environment(s):
+
+```bash
+sudo tar -cvpzf juju-client-$(date "+%Y%m%d-%H%M%S").tar.gz --exclude=.juju/local --exclude=.juju/local2 .juju 
+```
+
+!!! Note: As mentioned previously, the files in this backup include the keys 
+Juju uses to connect to running environments. Anybody who has access to this 
+backup will be able to connect to and control your environments, so a further 
+step of encrypting this backup file may be advisable.
+ 
+### Restoring ~/.juju
+
+Restoring your Juju client settings and environments is a simple matter of 
+extracting the archive you created above.
+
+```
+tar xzf juju-yymmdd-hhmmss.tar.gz 
+```
+
+In the case of any local environment, this will also be restored providing the 
+actual service itself (LXC) has remained running.
+
+## The Juju state server (bootstrap node)
+
+Juju provides commands for recovering the Juju state server (bootstrap
+node) in case of failure. Included commands allow you to create, restore and 
+manage backup file containing the state server
+configuration, keys, and environment data. If the state server or
+its host machine later fails, you can create a new state server from the
+backup file. For environments with "High availability" enabled, also see the 
+[relevant sections below](#ha-(high-availability))
+
+### Creating a backup file
+
+Use the `create` command to create a new backup file:
+
+```bash
+juju backups create [-e ENV] [--quiet ] [--filename=FILENAME | --no-download]
+```
+
+The `create` command creates an archive file for the state server in an
+environment, along with metadata about that file.  By default the current
+environment is used. It may be explicitly identified by the -e option
+or the `JUJU_ENV` shell environment variable.
+
+As a convenience, the state server stores both the file and the
+metadata (use `juju backups list` to see stored files), however, it is up to the
+user to download and safely store these files for when they are needed. 
+By default `juju backups create`will download the new backup file, though you
```

Comment:
I don't understand. The creation of a backup will download the new backup file? Should that be upload (to the state server)?